### PR TITLE
Update QUEST_JOBSTEP.tsv

### PR DESCRIPTION
**Pull request by Sourpusss**
Lines  3072 - 3138

Line 3091:  Just to be clear, these 3 Masters are "Amjinases?"  I'm assuming Amjinases is plural and just the Necro Master would be an Amjinas.

Line 3101: The second sentence is really odd, but I can't exactly tell what the context is (aside from the orb and exploration thing).  What is dangerous?  The mines?  The orb?  I changed the wording a bit to make it more fluid, but it's still kind of confusing.

Line 3128 (Line 3087 too): Not sure what "31_1" is?  I took out the second "the first Fletcher Master," as it's a bit redundant.

Line 3125: I'm like 98% sure the Scout Master in that map is female.

### CODE REVIEW

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -3070,13 +3070,13 @@ QUEST_JOBSTEP_20150730_003069	This was when the Coral Plain was still sea.{nl}I
...
 QUEST_JOBSTEP_20150730_003072	Good!{nl}I will teach you everything. 
-QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were to find it a little sooner.{nl}I will teach you the new formation as a reward. It's precise so it's much stronger.
-QUEST_JOBSTEP_20150730_003074	If you are a Doppelsoeldner, you should do something for the kingdom.{nl}I want to do it myself, but since it is being done quietly, I will be spotted if I do it myself.
-QUEST_JOBSTEP_20150730_003075	Thanks for saying so. Please find the envoy of Ramin General secretly.{nl}His last contact was from 36_2 of Pilgrim's way.
-QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyway, since you've become a member of Doppensoeldners, I will support you with my best.
-QUEST_JOBSTEP_20150730_003077	Ah, and please keep this secret to other people.{nl}I gotta find an other way to contact Ramin general.
+QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were able to find that a little sooner.{nl}I will teach you the new formation as a reward. It's precise, so it's much stronger.
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: `could` should probably be `would` in this situation

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003089	I have recently been researching eternal life.{nl}I am investigating various documents, but it will be the best to listen from those who are already living an eternal life.{nl}
+QUEST_JOBSTEP_20150803_003090	Those that have eternal life are called Amjinases. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
+QUEST_JOBSTEP_20150803_003091	Among the Amjinases, there is the Sculptor Tesla at West Siauliai Woods...{nl}The Necromancer Master at Mullers Passage, and the Chronomancer Master at Stele Road. I am counting on you.
+QUEST_JOBSTEP_20150803_003092	These Amjinases all seem to have different reasons behind their situations.{nl}I should look harder at each...{nl}
+QUEST_JOBSTEP_20150803_003093	Anyway, thank you for helping my research.{nl}In return, I'll teach you any remaining knowledge of what it is to be a Sorcerer.
+QUEST_JOBSTEP_20150803_003094	You've come to learn high level Chronomancer magic.{nl}But, right now I am helping Vaidotas from the Winterspoon Family, so I can't help you at the moment.{nl}
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: `high level Chronomancer magic.{nl}But, right`
to
`high level Chronomancer magic, {nl}but right`
for the sake of flow and connecting the two thoughts together (and also less starting sentences with "But," )

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003095	{nl}If you are okay, could you help me investigate the Crystal Mine?{nl}If so, I will finish up quickly and focus on helping you.
+QUEST_JOBSTEP_20150803_003096	With this orb, completely explore District 1 and 2 of the Crystal Mine.{nl}If you've done this already, then you just need to record it in the orb.
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: maybe `With this orb, explore all of District 1 and 2` instead? its up to you!

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
-QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me a favor.{nl}Return with this.
-QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family so I've requested it to him in advance. Tell him that I've sent you.
-QUEST_JOBSTEP_20150803_003105	Lucid... Actually have you met the Chronomancer Master well?
-QUEST_JOBSTEP_20150803_003106	The orb reads the past memories of its owner and records them.{nl}Explore all areas in District 1 and 2 in the Crystal Mine with this.{nl}
-QUEST_JOBSTEP_20150803_003107	When you use this orb after exploring all regions, it will read the memories of the possessor and record the past.{nl}If you had explored the regions in the past, then record it into the orb right away.{nl}
-QUEST_JOBSTEP_20150803_003108	Well done. {nl}It will take some time to check this so in the meantime, I will teach you new alchemy.
+QUEST_JOBSTEP_20150803_003098	The thoughts of the dead...All Necromancers want to possess them.{nl}Especially those regarding lost keepsakes.{nl}
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: keep the space after the ellipses, and consider connecting the last two sentences so this dialogue is less fragmented...
so maybe like:
`All Necromancers want to possess them, {nl}especially those`

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003099	According to what I found out, there's a strong grudge swirling around Videntis Shrine.{nl}If you could obtain whatever it is causing this grudge, I will open the door to a higher level of Necromancy.
+QUEST_JOBSTEP_20150803_003100	If you will be able to pick out this kind of object, you will become a stronger Necromancer.{nl}For now, I'll reward you if you can find one.
+QUEST_JOBSTEP_20150803_003101	Ah, you came at the right time. {nl}I am investigating Crystal Mine, but it will be dangerous if we leave it as is, so I want to take care of it quickly.{nl}
+QUEST_JOBSTEP_20150803_003102	If you could help me, I think the investigation would be done sooner.{nl}And it will be also a great chance to test your skills as an Alchemist.
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: should remove `And` from the start of the second sentence, it's a bit unnecessary

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me for a favor.{nl}Return with this.
+QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family, so I've requested it from him in advance. Tell him that I've sent you.
+QUEST_JOBSTEP_20150803_003105	Lucid...Actually have you met the Chronomancer Master well?
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Put the space back after the ellipses here too

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
 QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in West Siauliai Woods, the Necromancer Master in Mullers Passage, and the Chronomancer Master in Stele Road.
 QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Return to the Sorcerer Master.
-QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes from which you could feel strong regrets.
-QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepsake with a stronger grudge. Obtain the keepsakes that you can feel strong regrets from Videntis Shrine.
-QUEST_JOBSTEP_20150803_003117	You've collected the keepsakes with strong regrets. Return to the Necromancer Master.
+QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes that emit a strong sense of regret.
+QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants keepsakes with a stronger grudge. Obtain the menacing keepsakes from Videntis Shrine.
+QUEST_JOBSTEP_20150803_003117	You've collected the keepsakes, of which have a heavy aura about them. Return to the Necromancer Master.
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: maybe just have it as `You've collected the keepsakes that have a heavy aura about them.` ?

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003125	The Rogue Master wants to give a surprise present to the Scout Master. Find the Scout Master in Gateway of Great King and place the present secretly in her pocket.  It seems that this is fairly important for him - they must have been close at one point.
+QUEST_JOBSTEP_20150803_003126	You've succeeded in placing the present without being detected. Return to the Rogue Master.
 QUEST_JOBSTEP_20150803_003127	Search for the traces of Lydia Schaffen
-QUEST_JOBSTEP_20150803_003128	The Fletcher Master wants to look for the bow which the first the Fletcher Master made for Lydia Schaffen. Find the bow of the first Flectcher Master from 31_1 of Canyon Area.
+QUEST_JOBSTEP_20150803_003128	The Fletcher Master wants you to look for the bow that the first Fletcher Master made for Lydia Schaffen. Find the bow from 31_1 of Canyon Area.
 QUEST_JOBSTEP_20150803_003129	Give them to the Fletcher Master
-QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the Fletcher Master.
+QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the Fletcher Master that exists today.
```
Comment (sususuyo) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: maybe the last bit could just work as `Return to the current Fletcher Master` ?

**Comment by sususuyo** [comment]
Good work!! :+1: 
I left some suggestions and things that should probably be touched up, I hope you'll consider em!

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -3070,13 +3070,13 @@ QUEST_JOBSTEP_20150730_003069	This was when the Coral Plain was still sea.{nl}I
...
-QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyway, since you've become a member of Doppensoeldners, I will support you with my best.
-QUEST_JOBSTEP_20150730_003077	Ah, and please keep this secret to other people.{nl}I gotta find an other way to contact Ramin general.
+QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were able to find that a little sooner.{nl}I will teach you the new formation as a reward. It's precise, so it's much stronger.
+QUEST_JOBSTEP_20150730_003074	If you are a Doppelsoeldner, you should do something for the kingdom.{nl}I want to do it myself, but since it has to be done quietly, I don't trust my ability to take care of the situation.
+QUEST_JOBSTEP_20150730_003075	Thanks for saying so. Please find the envoy of Ramin General as secretly as possible.{nl}His last contact was from 36_2 of Pilgrim's way.
+QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyway, since you've become a member of Doppelsoeldners, I will support you with my all.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Sad. -> What a pity.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -3070,13 +3070,13 @@ QUEST_JOBSTEP_20150730_003069	This was when the Coral Plain was still sea.{nl}I
...
+QUEST_JOBSTEP_20150730_003077	Ah, and please keep this a secret.{nl}I gotta find an other way to contact Ramin General.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: I gotta find an other way -> I have to find another way
Ramin General -> General Ramin.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -3070,13 +3070,13 @@ QUEST_JOBSTEP_20150730_003069	This was when the Coral Plain was still sea.{nl}I
...
 QUEST_JOBSTEP_20150730_003078	If you've explored the area completely in the past, just record it in the orb.{nl}After exploring all the areas in Crystal Mine, use the orb to record your past.
-QUEST_JOBSTEP_20150730_003079	Explore all the regions in the 1st and 2nd districts with this orb.{nl}If you've explored the areas completely in the past, just record it in the orb.{nl}After exploring all the area, use the orb to record your past.
+QUEST_JOBSTEP_20150730_003079	Explore all the regions in the 1st and 2nd districts with this orb.{nl}If you've completely explored the areas already, just record it in the orb.{nl}After obtaining a 100% exploration rate for these areas, use the orb to record your past.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Explore all the regions in the 1st and 2nd districts with this orb -> Explore all the areas in District 1 and 2 with this orb.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 12]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
 QUEST_JOBSTEP_20150730_003084	Talk to the Squire Master
 QUEST_JOBSTEP_20150730_003085	Collected all the pieces of Legwyn Family Seal. Return to the Squire Master.
 QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Bradislovas
-QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas and use them as a proof of the victory. Look for the objects of Bradislovas that went under the sea at 32_1 of Coral Plain which once was sea. 
+QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas, and use them as a proof of the victory. Look for Bradislovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: _The_ Corsair Master 

Bradislovas -> Vladslovas (can you mass replace this, please?)

of the great pirate, Bradislovas -> of the Great Pirate Vladslovas

32_1 of Coral Plain -> Coral Plains 32_1

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 13]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
 QUEST_JOBSTEP_20150730_003084	Talk to the Squire Master
 QUEST_JOBSTEP_20150730_003085	Collected all the pieces of Legwyn Family Seal. Return to the Squire Master.
 QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Bradislovas
-QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas and use them as a proof of the victory. Look for the objects of Bradislovas that went under the sea at 32_1 of Coral Plain which once was sea. 
+QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas, and use them as a proof of the victory. Look for Bradislovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up. 
 QUEST_JOBSTEP_20150730_003088	It seems that the envoy is dead. Return to Doppensoeldner Master and report.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Doppensoeldner -> Doppelsoeldner

-> Return and report to the Doppensoeldner Master.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 14]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
-QUEST_JOBSTEP_20150803_003093	Anyway, thank for helping my research.{nl}As a return, I will teach you the last level of Sorcerer.
-QUEST_JOBSTEP_20150803_003094	You've come to learn high level magic of the Chronomancers.{nl}But, right now I am helping Vaidotas from the Winterspoon Family so I can't help you at the moment.{nl}
-QUEST_JOBSTEP_20150803_003095	{nl}If you are okay, could you help me investigating the Crystal Mine?{nl}If that's so, I will finish it quickly and help you.
-QUEST_JOBSTEP_20150803_003096	With this orb, explore all regions in District 1 and 2 in the Crystal Mine.{nl}If you had explored them completely in the past, then you just need to record it in the orb.
+QUEST_JOBSTEP_20150803_003089	I have recently been researching eternal life.{nl}I am investigating various documents, but it will be the best to listen from those who are already living an eternal life.{nl}
+QUEST_JOBSTEP_20150803_003090	Those that have eternal life are called Amjinases. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: The first sentence can be intepreted as if the the ones with eternal life are called "Amjinases", while it is "Amjinas".  I suggest changing it to: "the Amjinas".

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 15]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003091	Among the Amjinases, there is the Sculptor Tesla at West Siauliai Woods...{nl}The Necromancer Master at Mullers Passage, and the Chronomancer Master at Stele Road. I am counting on you.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: See above note.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 16]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003092	These Amjinases all seem to have different reasons behind their situations.{nl}I should look harder at each...{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: See above note.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv (hunk 2 above)
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: You've come to learn high level Chronomancer magic -> 

You've come to learn a higher level of our magic. 

or

You've come to learn a higher level of magic of the Chronomancers.

or

You've come to learn a higher level of Chronomancy.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 17]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
-QUEST_JOBSTEP_20150803_003107	When you use this orb after exploring all regions, it will read the memories of the possessor and record the past.{nl}If you had explored the regions in the past, then record it into the orb right away.{nl}
-QUEST_JOBSTEP_20150803_003108	Well done. {nl}It will take some time to check this so in the meantime, I will teach you new alchemy.
+QUEST_JOBSTEP_20150803_003098	The thoughts of the dead...All Necromancers want to possess them.{nl}Especially those regarding lost keepsakes.{nl}
+QUEST_JOBSTEP_20150803_003099	According to what I found out, there's a strong grudge swirling around Videntis Shrine.{nl}If you could obtain whatever it is causing this grudge, I will open the door to a higher level of Necromancy.
+QUEST_JOBSTEP_20150803_003100	If you will be able to pick out this kind of object, you will become a stronger Necromancer.{nl}For now, I'll reward you if you can find one.
+QUEST_JOBSTEP_20150803_003101	Ah, you came at the right time. {nl}I am investigating Crystal Mine, but it will be dangerous if we leave it as is, so I want to take care of it quickly.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: I am investigating Crystal Mine -> I am investigating **the** Crystal Mine

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 18]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003102	If you could help me, I think the investigation would be done sooner.{nl}And it will be also a great chance to test your skills as an Alchemist.
+QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me for a favor.{nl}Return with this.
+QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family, so I've requested it from him in advance. Tell him that I've sent you.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Winterspoon Family, -> Winterspoon family,

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 19]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
+QUEST_JOBSTEP_20150803_003106	The orb reads the past memories of its owner and records them.{nl}Completely explore District 1 and 2 in the Crystal Mine with this.{nl}
+QUEST_JOBSTEP_20150803_003107	When you use this orb after exploring completely, it will read the memories of the possessor and record the past.{nl}If you had explored the regions in the past, then record it into the orb right away.{nl}
+QUEST_JOBSTEP_20150803_003108	Well done. {nl}It will take some time to investigate this, so in the meantime, I will teach you new alchemy techniques.
 QUEST_JOBSTEP_20150803_003109	Eternal Amjinas [Sorcerer Advancement]
 QUEST_JOBSTEP_20150803_003110	Tell him that it will be better to listen directly
 QUEST_JOBSTEP_20150803_003111	I don't have guts for it
 QUEST_JOBSTEP_20150803_003112	Received enough balm. Restore the Statue of Goddess Vakarine in Fedimian and Statue of Goddess Ausrine in Klaipeda.
 QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in West Siauliai Woods, the Necromancer Master in Mullers Passage, and the Chronomancer Master in Stele Road.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Amjinases -> the Amjinas.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 20]
```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
...
 QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Return to the Sorcerer Master.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Amjinases -> the Amjinas.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv (hunk 8 above)
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: "that exists today" sounds odd. @sususuyo's suggestion is better in my opinion.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv (hunk 17 above)
Comment (Sourpusss) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Gotcha - I'm glad the anal-ness surrounding adding "the" infront of proper nouns has dropped off.  I'll keep that in mind from now on. (I just remember a long time back everyone was really weird about it).

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv (hunk 8 above)
Comment (Sourpusss) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: I was gonna go with current originally, but I put exists today to hint towards a larger gap of time.  I'll do current though.

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv (hunk 10 above)
Comment (Sourpusss) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: Oh wow, I didn't even spot "an other" - haha

**Comment by Sunaries** [comment]
By the way, we have a Discord where most contributors and dedicated contributors are present: https://discord.gg/0RoYSiES4f4zwAk8

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 21]
```diff
@@ -3066,73 +3066,73 @@ QUEST_JOBSTEP_20150730_003065	As the last head of the family, I am trying to rai
...
+QUEST_JOBSTEP_20150730_003087	The Corsair Master wants to look for the objects of the Great Pirate Vladslovas, and use them as a proof of the victory. Look for Vladslovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up. 
+QUEST_JOBSTEP_20150730_003088	It seems that the envoy is dead. Return and report to the Doppelsoeldner Master.
+QUEST_JOBSTEP_20150803_003089	I have recently been researching eternal life.{nl}I am investigating various documents, but it will be the best to listen from those who are already living an eternal life.{nl}
+QUEST_JOBSTEP_20150803_003090	Those that have eternal life are called Amjinas. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
+QUEST_JOBSTEP_20150803_003091	Among the Amjinas, there is the Sculptor Tesla at West Siauliai Woods...{nl}The Necromancer Master at Mullers Passage, and the Chronomancer Master at Stele Road. I am counting on you.
+QUEST_JOBSTEP_20150803_003092	These Amjinas all seem to have different reasons behind their situations.{nl}I should look harder at each...{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv:3092]: The Amjinas

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 22]
```diff
@@ -3066,73 +3066,73 @@ QUEST_JOBSTEP_20150730_003065	As the last head of the family, I am trying to rai
...
+QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Vladslovas
+QUEST_JOBSTEP_20150730_003087	The Corsair Master wants to look for the objects of the Great Pirate Vladslovas, and use them as a proof of the victory. Look for Vladslovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up. 
+QUEST_JOBSTEP_20150730_003088	It seems that the envoy is dead. Return and report to the Doppelsoeldner Master.
+QUEST_JOBSTEP_20150803_003089	I have recently been researching eternal life.{nl}I am investigating various documents, but it will be the best to listen from those who are already living an eternal life.{nl}
+QUEST_JOBSTEP_20150803_003090	Those that have eternal life are called Amjinas. I want you to ask them how they live eternal lives.{nl}It's not good for me to visit them as a master since we masters are living in different places.
+QUEST_JOBSTEP_20150803_003091	Among the Amjinas, there is the Sculptor Tesla at West Siauliai Woods...{nl}The Necromancer Master at Mullers Passage, and the Chronomancer Master at Stele Road. I am counting on you.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: there is the Sculptor Tesla -> there is Sculptor Tesla

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 23]
```diff
@@ -3066,73 +3066,73 @@ QUEST_JOBSTEP_20150730_003065	As the last head of the family, I am trying to rai
...
-QUEST_JOBSTEP_20150803_003092	Amjinases are indeed different each other.{nl}I should think more widely and find them out.{nl}
-QUEST_JOBSTEP_20150803_003093	Anyway, thank for helping my research.{nl}As a return, I will teach you the last level of Sorcerer.
-QUEST_JOBSTEP_20150803_003094	You've come to learn high level magic of the Chronomancers.{nl}But, right now I am helping Vaidotas from the Winterspoon Family so I can't help you at the moment.{nl}
-QUEST_JOBSTEP_20150803_003095	{nl}If you are okay, could you help me investigating the Crystal Mine?{nl}If that's so, I will finish it quickly and help you.
-QUEST_JOBSTEP_20150803_003096	With this orb, explore all regions in District 1 and 2 in the Crystal Mine.{nl}If you had explored them completely in the past, then you just need to record it in the orb.
+QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Vladslovas
+QUEST_JOBSTEP_20150730_003087	The Corsair Master wants to look for the objects of the Great Pirate Vladslovas, and use them as a proof of the victory. Look for Vladslovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: 32_1 of Coral Plain -> Coral Plains 32_1 (mass replace is useful)

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 24]
```diff
@@ -3066,73 +3066,73 @@ QUEST_JOBSTEP_20150730_003065	As the last head of the family, I am trying to rai
...
-QUEST_JOBSTEP_20150803_003091	Among the Amjinas, the whereabouts of the Sculptor Tesla at the West Siauliai Woods...{nl}the Necromancer Master at Mullers Passage and the Chronomancer Master at Stele Road. I am counting on you.
-QUEST_JOBSTEP_20150803_003092	Amjinases are indeed different each other.{nl}I should think more widely and find them out.{nl}
-QUEST_JOBSTEP_20150803_003093	Anyway, thank for helping my research.{nl}As a return, I will teach you the last level of Sorcerer.
-QUEST_JOBSTEP_20150803_003094	You've come to learn high level magic of the Chronomancers.{nl}But, right now I am helping Vaidotas from the Winterspoon Family so I can't help you at the moment.{nl}
-QUEST_JOBSTEP_20150803_003095	{nl}If you are okay, could you help me investigating the Crystal Mine?{nl}If that's so, I will finish it quickly and help you.
-QUEST_JOBSTEP_20150803_003096	With this orb, explore all regions in District 1 and 2 in the Crystal Mine.{nl}If you had explored them completely in the past, then you just need to record it in the orb.
+QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Vladslovas
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: great pirate, Vladslovas -> Great Pirate Vladslovas

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 25]
```diff
@@ -2967,8 +2967,8 @@ QUEST_JOBSTEP_20150729_002966	I heard there are kingdom soldiers who went lost a
...
-QUEST_JOBSTEP_20150729_002970	The person I am trying to meet is the envoy of Ramin General.{nl}I didn't hear anything from him since we had contacted each other at 36_2 of Way of Pilgrims.{nl}Go there and find out whether something is happening.
-QUEST_JOBSTEP_20150729_002971	Damn... I don't want to believe it, but accroding to what you just told me, it seems that he died due to the attacks from the monsters.{nl}I guess I should contact Ramin General with another method.{nl}Ah, and please keep this secret to other people.{nl}
+QUEST_JOBSTEP_20150729_002970	The person I am trying to meet is the envoy of General Ramin.{nl}I didn't hear anything from him since we had contacted each other at 36_2 of Way of Pilgrims.{nl}Go there and find out whether something is happening.
+QUEST_JOBSTEP_20150729_002971	Damn... I don't want to believe it, but accroding to what you just told me, it seems that he died due to the attacks from the monsters.{nl}I guess I should contact General Ramin with another method.{nl}Ah, and please keep this secret to other people.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: accroding -> according

File: EnglishTranslation-master/QUEST_JOBSTEP.tsv [hunk 26]
```diff
@@ -2967,8 +2967,8 @@ QUEST_JOBSTEP_20150729_002966	I heard there are kingdom soldiers who went lost a
...
 QUEST_JOBSTEP_20150729_002969	There's something I am preparing right now so I should meet someone, but he can't be contacted lately.{nl}I want to go and check him myself, but since the task I am preparing is being progressed quietly, I can't go there since I will be spotted by lots of people.{nl}Can you go there for me?
-QUEST_JOBSTEP_20150729_002970	The person I am trying to meet is the envoy of Ramin General.{nl}I didn't hear anything from him since we had contacted each other at 36_2 of Way of Pilgrims.{nl}Go there and find out whether something is happening.
-QUEST_JOBSTEP_20150729_002971	Damn... I don't want to believe it, but accroding to what you just told me, it seems that he died due to the attacks from the monsters.{nl}I guess I should contact Ramin General with another method.{nl}Ah, and please keep this secret to other people.{nl}
+QUEST_JOBSTEP_20150729_002970	The person I am trying to meet is the envoy of General Ramin.{nl}I didn't hear anything from him since we had contacted each other at 36_2 of Way of Pilgrims.{nl}Go there and find out whether something is happening.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_JOBSTEP.tsv]: 36_2 of Way of Pilgrims -> Pilgrim's Way 36_2 (mass replace is useful)

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**